software/28.Gin/SETUP.md

```diff
@@ -0,0 +1,36 @@
+# Setup
+
+## 1. Go
+
+In this workshop, we will create an HTTP REST server using Go.
+First of all, make sure you have **Go 1.14** or higher installed, allowing you to use the Gin Web Framework.
+
+If you have the right version installed, you should have something similar:
+```shell
+go version
+> go version go1.18.2 linux/amd64 # OK
+```
+
+> The language should NOT be an issue in this workshop, if you're struggling with Go, do not hesitate to ask for help !
+
+## 2. Postman
+
+We will be using `postman` to test out our routes.
+
+- [Postman](https://www.postman.com/downloads/)
+
+## 3. Gin
+
+Once everything is installed, you have to download the Gin package.
+Go at the root of your repo and run this commands to init a go project:
+```shell
+go mod init workshop-gin
+```
+
+Afterwards you can follow the official [documentation](https://github.com/gin-gonic/gin#installation) to install Gin correctly.
+>Once again don't hesitate if you a$have any issue during the installation ! :smile:
+
+
+**Now everything should work properly you can start the workshop !**
+
+[Go back to the exercises](./README.md)
```

Comment:
I think there is multiple issues on this this part. Firstly, you should not init the project in the gin installation section. I think you should add it in the step 0. Secondly you have two options, the first is to explain to create a main.go and to put import on it or secondly you can explain that in the step 0 part (i think its the best way).